components/x-live-blog-post/readme.md

```diff
@@ -1,14 +1,14 @@
-# x-content-block
+# x-live-blog-post
 
-This module displays a block of content with an id, title, body and timestamp. Main use case is the new version of live blogs, although we don't want to limit its usage to it.  
+This module displays a live blog post with an id, title, body and timestamp.  
```

Comment:
Pedantically, this sounds like the post displays the ID, which it doesn't. Although it does display the writer's name and social media links, which might be worth mentioning.

This is a side note and unrelated to the purpose of this PR, but I am raising it in case you want to do a small follow-up PR.

Reply:
Will do that. Thank you!